### DISCUSSION TITLE
Block 'Neueste Beiträge' -> optionale Einstellungen

### DISCUSSION THREAD

**Discussion by Nachtmensch87** [codeispoetry/sunflower#146] [Ideen und Wünsche]
Den Sunflower-Block 'Neueste Beiträge' um folgende optionale Einstellungen erweitern:

1. Anzahl der maximal anzuzeigenden Beiträge festlegen
2. Kategorie festlegen, sodass nur Beiträge der definierten Kategorie angezeigt werden
3. individuelle 'Überschrift' festlegen + Option keine anzuzeigen

So könnte man z.B. individuelle 'Presseseiten' etc. gestalten. Man kann sich ja alle Pressemitteilungen anzeigen lassen über eine entsprechende Kategorie, jedoch ist dann die Seite (m.W. nach) nicht individuell zu gestalten.

**Comment by codeispoetry** [comment]
Das ist nun alles umgesetzt.
